src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -1,467 +1,482 @@
-// Copyright (c) Microsoft Corporation.

```

Comment:
Looks like this entire file wasn't actually changed either.

Reply:
There was weirdness at play here. I was attempting to fix a merge conflict in this file (something about snapping to a grid)  which I did, but for some reason Github thought I changed the entire file.